src/components/Profile.js

```diff
@@ -1,187 +1,210 @@
-                            <span><b>Email: </b>{userinfo.user.email}</span><br />
-                            <span><b>Role: </b>{userinfo.user_type}</span><br />
-                            <span><b>Zulip ID: </b>{userinfo.zulip_id}</span>
-                        </div>
-                    }
-                    </div>
-                    )
-                : 
-                <>
-                {
-                    this.props.edit === true ?
-                    <>
-                    <Message
-                        warning
-                        header='Important: Regarding Zulip ID'
-                        list={[
-                        'You can go the Zulip screen and click on your name from the user list on the right. On the URL, you will see a number after user. That is your Zulip user ID.',
-                        'Please enter this correctly as this will be used to get your Zulip stats, and is not changeable in future.',
-                        'If you have not signed up on Zulip yet, sign up first and then enter this information!',
-                        ]}
-                    />
-                    <Form> 
-                        <Form.Input
-                            name="name"
-                            value={this.state.name}
-                            onChange={this.onChange}
-                            label='Name'
-                            placeholder='Enter your username...' 
-                            fluid
-                            required
-                        />
-                        <Form.Input
-                            name="zulip_id"
-                            value={this.state.zulip_id ? this.state.zulip_id : userinfo.zulip_id}
-                            onChange={this.onChange}
-                            label='Zulip User ID'
-                            placeholder='Enter your Zulip user ID...'
-                            fluid
-                            required
-                        />
-                        <span></span>
-                        <Form.Input
-                            name="role"
-                            value='student'
-                            label='Role' 
-                            fluid
-                            readOnly
-                        />
-                        <Form.Button
-                            disabled={!this.state.name || !this.state.zulip_id}
-                            onClick={this.submitInfo}
-                        >
-                            SUBMIT
-                        </Form.Button>
-                    </Form>
-                    </>
-                    : <span>You have not filled in your basic information till now.</span>
-                }
-                </>
-            }
-            </>            
-        )
+  render() {
+    const { userinfo } = this.props;
+    {
+      userinfo && userinfo.length === 1
```

Comment:
@jalajcodes Please create an issue for this to remove this warning also.

Reply:
Okay, I'll do that.